styles/framework/common/framework.less

```diff
@@ -19,11 +19,6 @@ body {
     background: #fff;
 }
 
-.dx-viewport {
```

Comment:
NOTE:

framework.less = dx.spa.css in artifacts
dx.spa.css = framework CSS styles + reset CSS styles

In this PR, we remove SPA framework CSS styles only. We are going to make a decision on `dx.spa.css` in a separate PR.

Reply:
FI: https://github.com/DevExpress/DevExtreme/pull/7257